mayastor/src/bdev/nexus/nexus_bdev_children.rs

```diff
@@ -422,6 +424,38 @@ impl Nexus {
                     }
                 }
             }
+            DeviceEventType::AdminCommandCompletionFailed => {
+                let cn = &device;
+                for nexus in nexus::instances() {
+                    if nexus_channel::fault_nexus_child(nexus, cn) {
+                        info!(
+                            "{}: retiring child {} in response to admin command completion failure event",
+                            nexus.name,
+                            device,
+                        );
+
+                        let child_dev = device.to_string();
+                        Reactors::master().send_future(async move {
+                            // Error indicates it is already paused and another
+                            // thread is processing
+                            // the fault
```

Comment:
was this new line intentional?

Reply:
Doh, not sure how that crept in, will remove.